test_security_compliance.py

```
@@ -0,0 +1,157 @@
+            page_content = await page.content()
+            sensitive_patterns = ["password", "secret", "token", "key"]
+            for pattern in sensitive_patterns:
+                assert pattern not in page_content.lower(), f"Found sensitive data in page source: {pattern}"
+            
+            # Step 7: Test session security
+            print("\n🔒 Testing Session Security...")
+            
+            # Check secure cookie attributes
+            cookies = await context.cookies()
+            for cookie in cookies:
+                assert cookie.get("secure", False), "Cookie not marked as secure"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)